### PR TITLE
Add CLI to print both unigram embedding matrices

### PR DESCRIPTION
**Pull request by jbcdnr**

### CODE REVIEW

**Comment by jbcdnr** [comment]
Good sanity check indeed:
Matrices are the same but the C++ code prints the token `<PLACEHOLDER>` and its vector **first** while the python method returns it at the **last** row. I think index 0 is more consistent with other tokenizers, what do you think ?

**Comment by mpagli** [comment]
The Python code sorts the tokens by frequency before fetching the vectors, as the `<PLACEHOLDER>` token is artificial, his frequency is set to 0 and ends up at the end of the matrix. We could bring the two methods in sync by forcing the  `<PLACEHOLDER>` token to be first:

```python
def get_unigram_embeddings(self):
    vocab = list(self.get_vocabulary().items())
    vocab.sort(key=lambda x: x[1], reverse=True)
    vocab = [w for w, c in vocab]
    if vocab[-1] == '<PLACEHOLDER>':
        vocab[-1], vocab[0] = vocab[0], vocab[-1]
    return self.embed_sentences(vocab), vocab
```

**Comment by jbcdnr** [comment]
> The Python code sorts the tokens by frequency before fetching the vectors

Is there a reason for this? I would prefer to get the same order as in the FastText model.
I modified the python binding with my last commit in this direction.

File: src/sent2vec.pyx [hunk 1]
```diff
@@ -112,12 +113,10 @@ cdef class Sent2vecModel:
...
         freqs = list(self._thisptr.getUnigramsCounts())
         assert len(vocab) == len(freqs)
-        return {w:c for w, c in zip(vocab, freqs)}
+        return OrderedDict(zip(vocab, freqs))
 
     def get_unigram_embeddings(self):
-        vocab = list(self.get_vocabulary().items())
-        vocab.sort(key=lambda x: x[1], reverse=True)
-        vocab = [w for w, c in vocab]
+        vocab = [w for w, c in self.get_vocabulary().items()]
```
Comment (mpagli) [inline src/sent2vec.pyx]: no forcing the `<PLACEHOLDER>` token to be first? This is not modifying the order of tokens w.r.t. the previous commit. I'm fine keeping it that way, but I thought this is what you wanted to change.

Comment (jbcdnr) [reply src/sent2vec.pyx]: `<PLACEHOLDER>` is first in FastText by default if we don't reorder by frequency.

**Pull request merged**